pkgs/by-name/re/recaf/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  maven,
+  fetchFromGitHub,
+  jdk11,
+  makeBinaryWrapper,
+  libGL,
+  glib,
+  gtk3,
+  libXtst,
+  nix-update-script,
+}:
+let
+  version = "2.21.14";
+  jdk = jdk11;
+in
+maven.buildMavenPackage {
+  pname = "recaf";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "Col-E";
+    repo = "Recaf";
+    rev = "refs/tags/${version}";
+    hash = "sha256-aVOVfrACawYc9VlBZ6+IHiCDGPOsVLdAvwQHdByZ584=";
+  };
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
+
+  mvnHash = "sha256-MPPZXQilmMaj8n5SXP7fG1Q95pH5Ydu7jU+KRS9beO8=";
+  mvnJdk = jdk; # Required for tests
+
+  # Disable flaky tests
+  mvnParameters = "'-Dtest=!CompilerTest,!#testMaven+testMavenLoading'";
+
+  installPhase =
+    let
+      # Required by glassgtk3
+      libPath = lib.makeLibraryPath [
+        libGL
+        gtk3
+        glib
+        libXtst
+      ];
```

Comment:
Even GTK is `dlopen()`'d? Horror

Reply:
Welcome to Java — otherwise it complains about `glassgtk3`